docs/guide/adding-custom-widget-placeholder-content.md

```diff
@@ -0,0 +1,126 @@
+# Adding placeholders to custom widgets
+
+Custom widgets can also take advantage of non-schema field placeholders by adding code to the main module, preparing the widget template to show the custom placeholder content, and adding the placeholder content into an accessible folder.
+
+## Basic placeholder addition
+
+### Altering the `index.js` file
+
+ In order to add a custom image file, two options should be set within the main widget file, `placeholder: true` and the `placeholderImage` option with the value set to the file extension of the placeholder content without any `.` prefix.
```

Comment:
the module's `index.js` file

Reply:
(main widget file is in the eye of the beholder IMHO)

Reply:
Yeah, that is what I was struggling with. People like Greg seem to have an index.js file that just imports from a bunch of other files. Didn't want to alienate people who program like that.